onnxmltools/convert/common/utils.py

```diff
@@ -5,3 +5,15 @@
 # --------------------------------------------------------------------------
 
 from onnxconverter_common.utils import *  # noqa
+
+
+def hummingbird_installed():
```

Comment:
please move this func into https://github.com/microsoft/onnxconverter-common/blob/master/onnxconverter_common/utils.py.
 
this file will be deprecated.